core/modules/node/node.module

```diff
@@ -2108,7 +2044,7 @@ function node_page_default() {
     $nodes = node_load_multiple($nids);
     $build = node_view_multiple($nodes);
 
-    // 'rss.xml' is a path, not a file, registered in node_menu().
+    // 'rss.xml' is a view path, not a file, registered by views module.
     backdrop_add_feed('rss.xml', $site_config->get('site_name') . ' ' . t('RSS'));
```

Comment:
Perhaps do a `backdrop_valid_path('rss.xml')` as you did above here as well? In the event rss.xml has been deleted, we don't want a link to it in the meta tags.